modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/OrganizationResourceTest.java

```diff
@@ -862,6 +906,127 @@ private void _testGetOrganizationsPageWithFilter() throws Exception {
 		assertContains(organization2, (List<Organization>)page.getItems());
 	}
 
+	private void _testGetOrganizationWithNestedFields() throws Exception {
+		com.liferay.portal.kernel.model.Organization organization =
+			OrganizationTestUtil.addOrganization();
+
+		AccountEntry accountEntry1 = _addAccountEntry();
+
+		organizationResource.postAccountOrganization(
+			accountEntry1.getAccountEntryId(),
+			String.valueOf(organization.getOrganizationId()));
+
+		AccountEntry accountEntry2 = _addAccountEntry();
+
+		organizationResource.postAccountOrganization(
+			accountEntry2.getAccountEntryId(),
+			String.valueOf(organization.getOrganizationId()));
+
+		AssetEntry assetEntry = _assetEntryLocalService.fetchEntry(
+			_classNameLocalService.getClassNameId(
+				com.liferay.portal.kernel.model.Organization.class),
+			organization.getOrganizationId());
+
+		AssetTag assetTag = _assetTagLocalService.addTag(
+			RandomTestUtil.randomString(), assetEntry.getUserId(),
+			assetEntry.getGroupId(), RandomTestUtil.randomString(),
+			new ServiceContext());
+
+		_assetTagLocalService.addAssetEntryAssetTag(
+			assetEntry.getEntryId(), assetTag);
+
+		AssetVocabulary assetVocabulary = AssetTestUtil.addVocabulary(
+			TestPropsValues.getGroupId());
+
+		AssetCategory assetCategory = AssetTestUtil.addCategory(
+			TestPropsValues.getGroupId(), assetVocabulary.getVocabularyId());
+
+		_assetEntryAssetCategoryRelLocalService.addAssetEntryAssetCategoryRel(
+			assetEntry.getEntryId(), assetCategory.getCategoryId());
+
+		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+		_roleLocalService.addGroupRole(
+			organization.getGroupId(), role.getRoleId());
+
+		_resourcePermissionLocalService.setResourcePermissions(
+			TestPropsValues.getCompanyId(),
+			com.liferay.portal.kernel.model.Organization.class.getName(),
+			ResourceConstants.SCOPE_INDIVIDUAL,
+			String.valueOf(organization.getOrganizationId()), role.getRoleId(),
+			new String[] {ActionKeys.DELETE});
+
+		User user1 = UserTestUtil.addUser();
+		User user2 = UserTestUtil.addUser();
```

Comment:
same here, two users but only one related to the org and we check the the first one is returned and the second one not